requirements.txt

```diff
@@ -2,6 +2,6 @@ async_lru~=1.0
 galaxy_importer>=0.3.1
 jsonschema>=3.0
 packaging
-pulpcore>=3.15.0.dev
+pulpcore>=3.15.1
```

Comment:
I think it will break the CI even when you add "Required PR"

Reply:
I think it runs against 3.16.0.dev (which is >= 3.15.1) and thus works?

Reply:
yeah, you are right, I thought pulpcore master was 3.15.1.dev, but it is 3.16.0.dev